metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -11,12 +11,12 @@ import monaco.languages.Languages.Definition
 import monaco.editor.Editor.BuiltinTheme
 import monaco.editor.Editor.LineNumbersOption
 
-@js.native
+@ScalaJSDefined
```

Comment:
It's not necessary. I managed to avoid it by using Future[T], JSConverters.toJSPromise and a case to monaco.Thenable[T].